src/components/layouts/header/Header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import logo from '../../../images/veraPDF-logo-400.png';
+import './Header.scss';
+
+function Header() {
+  return (
+    <header className="app-header">
+      <div className="header-inner">
```

Comment:
So many wrappers... Do we really need both? I understand why left and right are needed. But what `header-inner` is for?
BTW, probably Material UI have some navigation controls which are rendered to more semantical tags like `<nav>`? Maybe we could use them instead of maintaining that CSS on or side.

In general, I try to use as less "wrapper", "inner" elements as possible. Also it's nice to use more semantical tags instead of divs where possible (especially taking into account that we're working on a project related to Accessibility ;-) )

Reply:
Removed useless wrapper;
`<nav>` is related with menu / navigation element